homeworks/react-continued-3/tanjuha/src/components/lectures-list/lectures-list.jsx

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import './lectures-list.css';
+
+export default function LecturesList() {
+
+    const [data, setData] = useState([]);
+    const[hidden, setHidden] = useState(false);
+
+    useEffect(() =>{
+        axios
+            .get("https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json")
+            .then(result => setData(result.data.lectures))
+            .catch(error => {
+                console.error(error);
+            });
+    },[]);
+
+    return(
+            <table className= 'table'>
+                <caption>
+                    <h2>
+                        Lectures
+                    </h2>
+                    <button
+                        onClick={() => setHidden(!hidden)}>
```

Comment:
if your next value based on your current value you need to pass callback like `isHidden => !isHidden`